tcms/testruns/tests/test_admin.py

```diff
@@ -65,3 +65,10 @@ def test_admin_testrun_execution_status_color(self):
                 #92d400
             </span>
             '''))
+
+    def test_admin_testrun_execution_status_icon(self):
```

Comment:
method name is incorrect, also on line 59. There is no `testrun` here and `admin` and `execution_status` are clear from the file name and the class name. These are essentially columns in the changelist view.